checklist/README.md

```diff
@@ -0,0 +1,36 @@
+# Checklist for Newcomers of Sublab
+
+***
+
+## Register a Github Account
+- Pls refer to https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github to create your personal account on Github
+- Follow the CUHK SZ Sun lab: https://github.com/cuhk-haosun
+- After the administrator approves, add your personal information to the memberlist: https://github.com/cuhk-haosun/doc-lab-record/blob/main/misc/member.list.md
+
+***
+
+## Generate a SSH Key
+- Pls refer to https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent to generate a new ssh key
+- Add the ssh public key to your account
```

Comment:
also add public key to https://github.com/cuhk-haosun/doc-lab-record/blob/main/misc/member.list.md